docs/conf.py

```diff
@@ -168,7 +170,7 @@ def linkcode_resolve(domain, info):
     else:
         linespec = ""
 
-    fn = relpath(fn, start=dirname(SimPEG.__file__))
+    # fn = relpath(fn, start=dirname(SimPEG.__file__))
```

Comment:
Undo this so documentation has source code links

Reply:
Forgot to do this. Good catch. But I would like to fix it so it works when Python and the repo are on different drives in Windows. It's just a pain to remember to do this each time.